### DISCUSSION TITLE
Ban request

### DISCUSSION THREAD

**Discussion by objecttothis** [opensourcepos/opensourcepos#3661] [Team Posts]
Can we please enact a temporary (at minimum) ban on @mckaygerhard?  The user is often combative and critical which is not helping the process.  Often if (s)he would just say what they are saying nicely it would be fine, but there is often a jab in the messages.  The conversation in #1491 is the latest.

**Comment by daN4cat** [comment]
Welcome to our world... of the past few months. He's a troll who wants to show how good he is (doing nothing) to sell his stuff (I would never install it because you don't know what's inside, security!!).

Instead of banning, please all follow my approach. Make sure the user is banned from your user only and he doesn't get any of your replies notification and you don't get any notification of what he writes.
In this way he gets quite ignored and hopefully reduces the noise. I know it's called passive aggression but we avoid winds up too.

**Comment by objecttothis** [comment]
done.
